build.sbt

```diff
@@ -0,0 +1,58 @@
+import Dependencies._
+
+lazy val spark = "2.0.2"
+lazy val si = spark match {
+  case "2.2.0" => SparkInfo("0.10.4", "0.13.1")
+  case "2.1.0" => SparkInfo("0.10.4", "0.12")
+  case "2.0.2" => SparkInfo("0.10.3", "0.11.2")
```

Comment:
Two possible improvements

 1. Take the first three characters instead of the whole string, thus providing bug-fix-version-invariance to the py4j and breeze versions.

 2. Make `spark` configurable in some way. I don't know how to do that with sbt.